website/datadog.erb

```diff
@@ -34,6 +31,9 @@
             <li<%= sidebar_current("docs-datadog-resource-integration_pagerduty_service_object") %>>
               <a href="/docs/providers/datadog/r/integration_pagerduty_service_object.html">datadog_integration_pagerduty_service_object</a>
             </li>
+            <li<%= sidebar_current("docs-datadog-resource-logs-pipeline") %>>
```

Comment:
The pipeline order should have it's own documentation page, as it's a separate resource. The documentation of the pipeline should reference it where necessary.